src/main/java/org/jenkinsci/plugins/casc/PluginInstallConfigurator.java

```diff
@@ -0,0 +1,95 @@
+package org.jenkinsci.plugins.casc;
+
+import hudson.Extension;
+import hudson.PluginManager;
+import hudson.PluginWrapper;
+import hudson.ProxyConfiguration;
+import hudson.model.UpdateSite;
+import hudson.util.VersionNumber;
+import jenkins.model.Jenkins;
+import org.apache.commons.lang.StringUtils;
+import sun.misc.Version;
+
+import java.io.StringWriter;
+import java.util.*;
+
+/**
+ * Created by mads on 1/29/18.
+ */
+@Extension(ordinal = Double.MAX_VALUE)
+public class PluginInstallConfigurator implements RootElementConfigurator {
```

Comment:
This doesn't actually install plugins, so should be renamed `PluginConfigurator` to avoid confusion.